src/cleanupEquation.js

```diff
@@ -15,6 +21,23 @@ const cleanupEquation = (result) => {
   tempResult = closeParens(tempResult) + "";
   tempResult = addMultiplier(tempResult) + "";
 
-  return tempResult;
+  // Handling double equal
+  // If I only have sign + digit(s) + dot + digit(s)
+  const regex = /-{0,1}[0123456789]*(\.[0123456789]*){0,1}/g;
+  const matches = tempResult.match(regex) || [];
+  if (matches.length > 1 && matches[0] === tempResult) {
+    //I need to handle the double equal
+    if ("+-/*".includes(operate) && !isNaN(originalLastNum)) {
+      // finalResult = evaluate(result + operate + originalLastNum) + "";
+      try {
+        finalResult =
+          roundedResult(tempResult + operate + originalLastNum) + "";
```

Comment:
unsure why this code was moved into here. why would we want to run the calculation inside "cleanupEquation"?

Reply:
This was so long ago in the refactor that I'm struggling to remember...😅 but actually I'm pretty sure it was because I was trying out setting finalResult and then returning it separately instead of at once. But this is no longer the case.